src/page-modules/contact/components/form/time-selector.tsx

```diff
@@ -29,7 +29,10 @@ export default function TimeSelector({
       <Label>{t(label)}</Label>
       <TimeField
         value={parsedValue}
-        onChange={(change) => onChange(change.toString())}
+        onChange={(change) => {
+          if (change === null) return;
```

Comment:
Does this only apply to `null` values?

```suggestion
          if (!change) return;
```

Reply:
Good point. This should apply for both `null` and `undefined`.